crates/collab_ui/src/chat_panel.rs

```diff
@@ -335,67 +336,77 @@ impl ChatPanel {
         };
         let this = cx.view().clone();
 
-        v_flex()
-            .w_full()
-            .relative()
-            .overflow_hidden()
-            .when(!is_continuation_from_previous, |this| {
-                this.pt_3().child(
-                    h_flex()
-                        .text_ui_sm()
-                        .child(
-                            div().absolute().child(
+        let mentioning_you = message
+            .mentions
+            .iter()
+            .any(|m| m.1 == self.client.user_id().unwrap());
```

Comment:
This use of `.unwrap()` is most likely fine, but our ideal policy is to only panic in code paths that you can tell are unreachable *locally* (i.e. based on the immediate surrounding method).

Could you make this slightly more defensive as follows:

```suggestion
            .any(|m| Some(m.1) == self.client.user_id());
```

Reply:
Yeah, makes sense. Didn't know you could fix it like this, I'm kinda new to Rust.